src/workerd/io/tracer.c++

```diff
@@ -381,8 +405,9 @@ void WorkerTracer::setFetchResponseInfo(tracing::FetchResponseInfo&& info) {
 
   KJ_IF_SOME(writer, maybeTailStreamWriter) {
     auto& spanContext = KJ_UNWRAP_OR_RETURN(topLevelInvocationSpanContext);
-    writer->report(
-        spanContext, tracing::Return({KJ_ASSERT_NONNULL(trace->fetchResponseInfo).clone()}));
+    // TODO: completeTime may not yet be available at this time
```

Comment:
This may be addressed by moving fetch response reporting to the destructor (as with outcome), would have to update a couple of tests as this causes fetch response to be reported before the "worker" span. Lmk if you can see a more elegant approach.